dspace-api/src/main/java/org/dspace/scripts/DSpaceCommandLineParameter.java

```diff
@@ -0,0 +1,108 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.scripts;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * This class serves as a representation of a command line parameter by holding a String name and a String value
+ */
+public class DSpaceCommandLineParameter {
```

Comment:
This seems to be a very important processing class. I think it needs Unit Tests to ensure the various methods are parsing out parameters as expected.  I'm not seeing Unit Tests.